### PR TITLE
Vagrant

### PR DESCRIPTION
**Pull request by Netsensia**
I have added a Vagrant/Salt set up which will create an Ubuntu 14.04 VM with the dashboard running under Apache. Basic step-by-step bullets within a README in the vagrant folder.

### CODE REVIEW

**Comment by brettminnie** [comment]
Seems travis doesn't agree with it chris

**Comment by eddiejaoude** [comment]
Ignore Travis for `Spike` branches. It shouldn't run for those branches.

**Comment by eddiejaoude** [comment]
I will config Travis to ignore branches that begin with `spike*`

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,7 @@
 .DS_Store
 .idea
 .project
+.settings/
```
Comment (eddiejaoude) [inline .gitignore]: Did the ignore go in after the `.settings/` directory was added? 

Can you remove the files in `.settings/` directory please & the directory itself:

* .settings/org.eclipse.php.core.prefs
* .settings/org.eclipse.php.debug.core.Debug_Process_Preferences.prefs

*Keep `.settings/` in the ignore*

**Comment by eddiejaoude** [comment]
Looks good :+1: - just the one comment :smile:

File: .gitignore (hunk 1 above)
Comment (eddiejaoude) [inline .gitignore]: I have removed these files

**Comment by eddiejaoude** [comment]
I didn't delete the `branch`, as we might re-use to **merge** into another

**Pull request merged**